documentation/modules/adding-source-provider.adoc

```diff
@@ -100,7 +100,7 @@ ifdef::rhv[]
 * *Provider resource name*: Name of the source provider
 +
 include::snip-provider-names.adoc[]
-* *URL*: API endpoint of the {rhv-short} Manager on which the source VM is mounted
+* *URL*: API endpoint of the {rhv-full} ({rhv-short}) Manager on which the source VM is mounted, for example, `https://rhv-host-example.com/ovirt-engine/api`
```

Comment:
It seems that currently it appears as:
"API endpoint of the **Red Hat Virtualization (RHV) Manager**...."

 It's ok, but to be aligned with the UI, it's better to change to:

"API endpoint of the **Red Hat Virtualization Manager (RHVM)** ...."

```suggestion
 *URL*: API endpoint of the {rhv-full} Manager  ({rhv-short})M on which the source VM is mounted, for example, `https://rhv-host-example.com/ovirt-engine/api`
```

Reply:
Fxed.